### PR TITLE
Feature/jeremy2

### PR DESCRIPTION
**Pull request by YmerejRedienhcs**
first draft of my outline

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,9 @@ This repository includes resources for the "Automated Testing of Web Mapping App
 There are many types of automated testing for web applications, from isolated unit tests of a single JavaScript function to end-to-end tests that exercise your entire stack in multiple browsers. We'll discuss the strengths and weaknesses of the main types of tests and share how we decide what kinds of tests to write. We'll give an overview of the different tools that are available for each type of test, and demonstrate the tools we use to write and automatically execute tests as part of our process for continuously delivering quality software.
 
 ## Presenters
-[@tomwayson](https://github.com/tomwayson)
-[@Brooks Robertson](https://github.com/brooksjbr)
+- [@tomwayson](https://github.com/tomwayson)
+- [@Brooks Robertson](https://github.com/brooksjbr)
+- [@Jeremy Schneider](https://github.com/ymerejredienhcs)
 TODO: add yourself here however you want to be linked to
```
Comment (tomwayson) [inline README.md]: let's kill this line

**Review by tomwayson — COMMENTED** [review]
Looks great - just made one suggested change

File: README.md (hunk 1 above)
Comment (YmerejRedienhcs) [reply README.md]: Which line?

**Comment by tomwayson** [comment]
TODO

File: README.md (hunk 1 above)
Comment (YmerejRedienhcs) [reply README.md]: Ah, ok, when I saw this it was on a device that didn't display it as clearly,  Thanks.

**Comment by YmerejRedienhcs** [comment]
@tomwayson I have removed the TODO line.

**Pull request merged**